samples/mini-games/src/godotMain/kotlin/example/TestingClass.kt

```diff
@@ -77,10 +78,11 @@ class TestingClass : Object() {
     var variantTest = Variant("test")
 
     @RegisterProperty
-    lateinit var resourceVisibleInEditor: Resource
+    lateinit var resourceVisibleInEditor: CustomResource //should be registered as Texture as Custom resource derives from texture
```

Comment:
I don't think this is correct example, adding a new resource type requires adding support to the engine. Plus we don't know how to instanstiate `CustomResource` at runtime if it has other parameters.

Reply:
But then we have to actively prevent it. I'll change it this evening

Reply:
Did it now ^^